pokemon-minisite/src/app/components/history/history.component.html

```diff
@@ -0,0 +1,17 @@
+<body>
+  <ng-container *ngIf="searchHistory.length > 0; else noHistory">
+    <main class="box">
+      <h2>Search History</h2>
+      <div class="inputBox">
+        <div *ngFor="let history of searchHistory; let i = index">
+          <h3 for="history">{{ i + 1 }}. {{ history }}</h3>
+        </div>
+      </div>
+        <button type="button" name="" (click)="goBack()">home</button>
```

Comment:
remove the type and the name

Reply:
removed name needed type to stay in order to keep home button style